src/realm/alloc.hpp

```diff
@@ -281,7 +281,7 @@ class Allocator {
     inline uint_fast64_t get_storage_version(uint64_t instance_version)
     {
         if (instance_version != m_instance_versioning_counter) {
-            throw LogicError(LogicError::detached_accessor);
+            throw StaleAccessor("Stale acessor version");
```

Comment:
```suggestion
            throw StaleAccessor("Stale accessor version");
```

Reply:
Would it be useful with info about instance_version and m_instance_versioning_counter ?